android/console.c

```diff
@@ -1817,6 +1817,19 @@ do_gsm_report( ControlClient  client, char*  args )
     return 0;
 }
 
+static int
+do_gsm_timezone( ControlClient  client, char  *args)
+{
+    if (!args) {
+        control_write( client, "KO: missing argument, try 'gsm timezone <+/-tzdiff>'\r\n" );
+        return -1;
+    }
+
+    int tzdiff = atoi(args);
```

Comment:
`atoi()` doesn't have error reporting, e.g. if the string does not begin with an integer, it will return 0. IMO, we should add some check and returns error if the `args` is not a valid input.